### PR TITLE
Add vm backup & restore doc

### PR DESCRIPTION
**Pull request by guangbochen**
Add vm backup & restore doc

### CODE REVIEW

**Comment by guangbochen** [comment]
@catherineluse would be nice if u can help to review this PR, thanks

File: docs/backup-restore.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+1. Users can either choose to restore a new VM or replace an existing VM using this backup.
+
+## Restore a new VM using a backup
+1. Users can select to restore a new VM using a backup from the `Backups` page.
+1. Specify the new VM name and click `Create`.
+1. A new VM will be restored using the backup volumes and metadata, and users can access it from the `Virtual Machines` page.
+
+## Replace an Existing VM using a backup
+1. Users can replace an existing VM using the backup with the same VM backup target on the `Backups` page.
+1. The VM must exist and required to be in the turned-off status.
```
Comment (gitlawr) [inline docs/backup-restore.md]:
```suggestion
1. The VM must exist and is required to be in the powered-off status.
```

File: docs/backup-restore.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+| Endpoint | string | EndPoint is a hostname or an IP address |
+| BucketName | string | Name of the bucket |
+| BucketRegion | string | Region of the bucket |
+| AccessKeyID | string | AccessKeyID is like a user-id that uniquely identifies your account. |
+| SecretAccessKey | string | SecretAccessKey is the password to your account. |
+| Certificate | string | Paste the certificate if you want to use a self-signed SSL certificate of your s3 server |
+| VirtualHostedStyle | bool | Use virtual-hosted-style access only, e.g., Alibaba Cloud(Aliyun) OSS |
+
+## Create a VM backup
+1. Once the backupTarget is set, go to the `Virtual Machine` page.
```
Comment (gitlawr) [inline docs/backup-restore.md]:
```suggestion
1. Once the backup target is set, go to the `Virtual Machines` page.
```

File: docs/backup-restore.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+## Restore a new VM using a backup
+1. Users can select to restore a new VM using a backup from the `Backups` page.
+1. Specify the new VM name and click `Create`.
+1. A new VM will be restored using the backup volumes and metadata, and users can access it from the `Virtual Machines` page.
+
+## Replace an Existing VM using a backup
+1. Users can replace an existing VM using the backup with the same VM backup target on the `Backups` page.
+1. The VM must exist and required to be in the turned-off status.
+1. Optional) Users can choose either delete the previous volumes or retain them, default to delete all previous volumes.
```
Comment (gitlawr) [inline docs/backup-restore.md]:
```suggestion
1. (Optional) Users can choose to either delete the previous volumes or retain them, default to delete all previous volumes.
```

**Comment by guangbochen** [comment]
@gitlawr updated

**Review by gitlawr — APPROVED** [review]
LGTM

**Pull request merged**